test/jest/integration-capture.spec.ts

```diff
@@ -0,0 +1,125 @@
+import IntegrationCapture from "../../src/integrationCapture";
+import { deleteAllCookies } from "./utils";
+
+describe('Integration Capture', () => {
+    describe('constructor', () => {
+        it('should initialize with clickIds as undefined', () => {
```

Comment:
I go back and forth with saying this should be undefined vs it being defined with a default value of {}.  But not a hill i'm going to die on :)